webhook/icinga.go

```diff
@@ -118,7 +118,6 @@ func createServiceData(hostname string,
 	serviceVars["keep_for"] = config.KeepFor
 	serviceVars = mapIcingaVariables(serviceVars, alert.Labels, "label_", l)
 	serviceVars = mapIcingaVariables(serviceVars, alert.Annotations, "annotation_", l)
-	serviceVars = addStaticIcingaVariables(serviceVars, config.StaticServiceVars, l)
```

Comment:
This should stay in, otherwise we break the static icinga variables feature completely.